packages/headless/src/features/commerce/facets/field-suggestions-order/field-suggestions-order-slice.ts

```diff
@@ -0,0 +1,12 @@
+import {createReducer} from '@reduxjs/toolkit';
+import {fetchQuerySuggestions} from '../../query-suggest/query-suggest-actions';
+import {getFieldSuggestionsOrderInitialState} from './field-suggestions-order-state';
+
+export const fieldSuggestionsOrderReducer = createReducer(
+  getFieldSuggestionsOrderInitialState(),
+  (builder) => {
+    builder.addCase(fetchQuerySuggestions.fulfilled, (_, action) => {
+      return action.payload.fieldSuggestionsFacets ?? [];
```

Comment:
No validation?

Reply:
What do you mean? The `fieldSuggestionsFacets` are what we want as state :eyes: